linkup-cli/src/status.rs

```diff
@@ -22,11 +23,38 @@ struct SessionStatus {
 #[derive(Deserialize, Serialize)]
 struct ServiceStatus {
     name: String,
-    status: String,
+    status: ServerStatus,
     component_kind: String,
     location: String,
 }
 
+#[derive(Deserialize, Serialize, PartialEq)]
+enum ServerStatus {
+    Ok,
+    Error,
+    Timeout,
+}
+
+impl ServerStatus {
+    fn colored(&self) -> ColoredString {
+        match self {
+            ServerStatus::Ok => "ok".green(),
+            ServerStatus::Error => "error".red(),
+            ServerStatus::Timeout => "timeout".yellow(),
```

Comment:
Nice! Two suggestions

1. Timeout is the "regular" failure when the server isn't started, it could also be "red"?
1. We got some colour blindness feedback recently - red/green is apparently the worst (even though it also tends to mean something 😅 ). Maybe just green/yellow instead?

<img width="427" alt="Screenshot 2023-05-16 at 09 29 35" src="https://github.com/mentimeter/linkup/assets/7828615/275a9072-bd1c-4c81-8c91-a6d9a6327646">

Reply:
Oh, that is a very good point. What do you think of this?
- ok -> blue
- error -> yellow
- timeout -> yellow

https://blog.datawrapper.de/colorblindness-part2/#Colorblind-safe-color-palettes

Reply:
Nice! Let's do it